components/services/bat_ads/BUILD.gn

```diff
@@ -1,7 +1,8 @@
 static_library("lib") {
   visibility = [
     "//brave/utility:*",
-    "//brave/test:*"
+    "//brave/test:*",
+    "//chrome/utility:*"
```

Comment:
why is this needed?

Reply:
yea, this doesn't make any sense here because there is no dep on this in `//chrome/utility`

Reply:
We do have this dependency in current (non brave-core) Android implementation.

Reply:
it doesn't exist in brave-core or this PR

Reply:
you should replace it with `//brave/utility` and then you won't have to change this visibility list

Reply:
Removed